### PR TITLE
fixes #6019 return to same bootstrap tab for compute resource after VM/Image create/update/delete

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
solution found at http://stackoverflow.com/questions/7862233/twitter-bootstrap-tabs-go-to-specific-tab-on-page-reload

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -95,6 +95,17 @@ function onContentLoad(){
     });
   });
 
+  // redirect to bootstrap tab #name
+  var hash = document.location.hash;
+  var prefix = "tab_";
+  if (hash) {
+      $('.nav-tabs a[href='+hash.replace(prefix,"")+']').tab('show');
```
Comment (domcleal) [inline app/assets/javascripts/application.js]: nitpick, 2-space indentation

File: app/controllers/compute_resources_vms_controller.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ def create
 
     if (vm = @compute_resource.create_vm params[:vm])
       @compute_resource.start_vm(vm.identity) if params[:vm][:start]=='1'
-      process_success :success_redirect => compute_resource_vms_path(@compute_resource)
+      process_success :success_redirect => compute_resource_path(@compute_resource) + "/#vms"
```
Comment (domcleal) [inline app/controllers/compute_resources_vms_controller.rb:25]: nitpick, did you want to add the tab_ prefix to these?

**Comment by domcleal** [comment]
There's a strange behaviour when you click one of the profiles under the compute profiles tab, after having got #images or something in the URL, as it switches back to the images tab when the AJAX for the compute profile reloads.  Maybe we need to clear the hash in the address bar on tab switch?

**Comment by domcleal** [comment]
Please update the commit message, should be #6019.

**Comment by isratrade** [comment]
commit msg updated.

**Comment by isratrade** [comment]
@ehelms, @waldenraines, can you take a look at the bug that @domcleal mentioned above where the AJAX causes a switch in the "redirect to tab". I encountered it a different way. I clicked on "Virtual Machines" and got  a spinner with  Loading virtual machines information.  Then I clicked on 'Compute profiles' tab and edited a compute profile. When I clicked submit, instead of returning to 'Compute profiles' tab, it returns to "Virtual Machines" tab with the spinner / message.

**Comment by waldenraines** [comment]
@isratrade I don't have a way to easily get to the same state but it sounds like @domcleal is correct about the anchor.

**Comment by isratrade** [comment]
closing. will re-open later

**Comment by isratrade** [comment]
Rebased for @tbrisker to look at.

**Pull request closed without merging**